docker-compose.yml

```diff
@@ -163,17 +165,6 @@ services:
             ca.mcgill.a11y.image.handler: enable
         volumes:
             - sc-store:/tmp/sc-store
-    generic-tts-handler:
```

Comment:
Why did you delete this?

Reply:
If you look at the lines above, there is a carbon copy of the ```generic-tts-handler```. I was not able to build the container when there were two processes named the same thing. If there is some purpose for there being two then I can add it back in.

Reply:
Oh no in that case don't worry. It only shows me the changes by default so it looked like you were deleting a whole thing...weird because we aren't having this problem in main as far as I can tell!

Reply:
See lines 157-167. 

I don't know how to link it because it's not part of the diff so it isn't giving me an option to comment on the lines.